source/jsonify.d.ts

```diff
@@ -0,0 +1,54 @@
+import {JsonPrimitive} from './basic';
+
+type NotJsonable = ((...args: any[]) => unknown) | undefined;
+
+/**
+Transform a type to a type that is assignable to the `JsonValue` type.
+
+@remarks
+
+An interface cannot be structurally compared to `JsonValue` because an interface can be re-opened to add properties that may not be satisfy `JsonValue`.
+
+Example:
+
+interface Geometry {
```

Comment:
The code should be wrapped in a proper code block. See any of the other types in here.

Reply:
See 903f94c72d6892788018fc14ad50cece7c551486